contracts/truefi/LoanToken.sol

```diff
@@ -301,6 +301,9 @@ contract LoanToken is ILoanToken, ERC20 {
      * @param _amount amount of currencyToken to repay
      */
     function repay(address _sender, uint256 _amount) external override onlyAfterWithdraw {
+        if (_amount > debt) {
```

Comment:
Revert in this case. It's better to throw mistake than quitely fix it and lead to user's confusion.

Reply:
I agree. Especially considering the fact, that this amount will not move unexpectedly and frontend app can fetch it and pass accurately.